go/vt/vtctl/grpcvtctldserver/topo.go

```diff
@@ -29,9 +34,65 @@ import (
 	"vitess.io/vitess/go/vt/vterrors"
 
 	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
+	vschemapb "vitess.io/vitess/go/vt/proto/vschema"
 	"vitess.io/vitess/go/vt/proto/vtrpc"
 )
 
+// decodeContent uses the filename to imply a type, and proto-decodes
+// the right object, then echoes it as a string.
+//
+// This is copied from vtctl/topo. We cannot import vtctl/topo because of an import loop.
```

Comment:
Is there a place to refactor this method so both packages can use it?

Reply:
Added it to `topo` because adding it to `topoproto` would introduce an import cycle.

Reply:
wait that doesn't make sense; this package imports `topoproto`, where is the import cycle?

Reply:
ohhh it's because of the `topo.GetExternalVitessClusterDir` got it

Reply:
Yeah! It uses a lot of the types from `topo`!